spec/util/vmdb-logger_spec.rb

```diff
@@ -10,31 +11,47 @@
       logger.log_hashes(hash)
 
       buffer.rewind
-      expect(buffer.read).to_not include("pa$$w0rd")
+      message = buffer.read
+      expect(message).to_not include("pa$$w0rd")
+      expect(message).to include("FILTERED")
```

Comment:
Can we change these specs to all compare against the expected message value directly?

That would make what `log_hashes` is doing a whole lot more clear.